apps/osvr_server.cpp

```diff
@@ -61,14 +61,14 @@ int main(int argc, char *argv[]) {
 
     optionsPositional.add("config", -1);
     optionsVisible.add_options()
-        ("config", "server configuration filename")
+        ("config", opt::value< std::vector<std::string> >(), "server configuration filename")
         ("help", "display this help message")
         ("verbose,v", "enable verbose logging")
         ("debug,d", "enable debug logging");
     optionsAll.add(optionsVisible);
 
     opt::variables_map values;
-    try {
+    try {        
```

Comment:
trailing whitespace - I can fix.

Reply:
Sorry about that, Visual Studio is always a PITA with this.

Reply:
I hear you. Recently edited code on a machine without the clang format extension installed. It was terrible.